internal/xds/translator/testdata/in/xds-ir/multiple-listeners-same-port.yaml

```diff
@@ -1,66 +1,70 @@
 http:
-- name: "first-listener"
-  address: "0.0.0.0"
-  port: 10080
-  hostnames:
-  - "foo.com"
-  tls:
-    serverCertificate: [99, 101, 114, 116, 45, 100, 97, 116, 97] # byte slice representation of "cert-data"
-    privateKey: [107, 101, 121, 45, 100, 97, 116, 97] # byte slice representation of "key-data"
-  routes:
-  - name: "first-route" 
-    destinations:
-    - host: "1.2.3.4"
-      port: 50000
-- name: "second-listener"
-  address: "0.0.0.0"
-  port: 10080
-  hostnames:
-  - "foo.net"
-  tls:
-    serverCertificate: [99, 101, 114, 116, 45, 100, 97, 116, 97] # byte slice representation of "cert-data"
-    privateKey: [107, 101, 121, 45, 100, 97, 116, 97] # byte slice representation of "key-data"
-  routes:
-  - name: "second-route" 
-    destinations:
-    - host: "1.2.3.4"
-      port: 50000
-- name: "third-listener"
-  address: "0.0.0.0"
-  port: 10080
-  hostnames:
-  - "example.com"
-  routes:
-  - name: "third-route" 
-    destinations:
-    - host: "1.2.3.4"
-      port: 50000
-- name: "fourth-listener"
-  address: "0.0.0.0"
-  port: 10080
-  hostnames:
-  - "example.net"
-  routes:
-  - name: "fourth-route" 
-    destinations:
-    - host: "1.2.3.4"
-      port: 50000
+  - name: "first-listener"
```

Comment:
can you revert the identation or was it generated ?

Reply:
looks like I run `Format Document` with vscode, let me revert this.